browse/src/main/kotlin/com/alfresco/content/browse/tasks/detail/TaskDetailFragment.kt

```diff
@@ -145,6 +160,11 @@ class TaskDetailFragment : Fragment(), MavericksView, EntryListener {
             binding.tvAssignedValue.text = dataObj.assignee?.name
             binding.tvStatusValue.text = if (dataObj.endDate == null) getString(R.string.status_active) else getString(R.string.status_completed)
             binding.tvIdentifierValue.text = dataObj.id
+
+            if (dataObj.endDate != null) {
```

Comment:
create a val to isCompleted to check the condition instead to check everywhere dataObj.endDate.
In the future, If we get the direct status of the task then doesn't need to replace the everywhere. We shall change in one place.

Reply:
Done